templates/CRM/Admin/Page/Options.tpl

```diff
@@ -156,7 +156,7 @@
     </div>
 {/if}
     <div class="action-link">
-      {if $isLocked ne 1}
+      {if !empty($isLocked) and $isLocked ne 1}
```

Comment:
I think this is wrong I think 0 would resolve to TRUE  before and FALSE after

Reply:
Thanks Eileen good pick up changed it to isset now